[ADD] Inventory: product catalog (#10983)
odoo/documentation: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/product_catalog.rst

===============
Product catalog
===============

.. |SO| replace:: :abbr:`SO (sales order)`
.. |SOs| replace:: :abbr:`SOs (sales orders)`
.. |BoM| replace:: :abbr:`BoM (bill of materials)`
.. |RfQ| replace:: :abbr:`RfQ (request for quotation)`

The product *catalog* is a feature integrated with any Odoo app that allows users to add products or
components to an order. This includes the **Inventory**, **Manufacturing**, **Sales**, **Purchase**,
and **Repairs** apps, among others.

The product catalog can be accessed from the first tab of a quotation, request for quotation (RfQ),

> order, or bill of materials (BoM) form, and opens in a new page when selected. The catalog displays

Reviewer: ```suggestion
sales order, or bill of materials (BoM) form, and opens in a new page when selected. The catalog displays
```

Author: This includes other order types, in addition to sales orders.

products and components in a user-friendly, POS-style format, from which they can be selected and
added to forms.

The product catalog simplifies the creation of new sales orders (SOs), purchase orders (POs),
manufacturing orders (MOs), bill of materials (BoMs), and more, by providing a visual interface
through which products and components can be quickly selected.

Use product catalog
===================

To use the product catalog, begin by creating or opening a quotation, |RfQ|, order, or |BoM| to
which products or components can be added. For example, create a new sales quotation by navigating
to :menuselection:`Sales app`, and clicking :guilabel:`New`.

On the form (quotation, |RfQ|, order, |BoM|), make sure the first tab on the bottom is selected.
Depending on the form being configured, this tab may be titled *Order Lines*, *Components*,
*Products*, or *Parts*.

> On the first blank line of the tab, click the :guilabel:`Catalog` button to open the catalog in a

Reviewer: ```suggestion
On the first blank line of the tab, click the :guilabel:`Catalog` link to open the catalog in a
```

new page.

.. image:: product_catalog/catalog-button.png
   :align: center
   :alt: The "Catalog" button on the "Order Lines" tab of a sales quotation.

The product catalog displays a card for each product added to Odoo. Each card displays a few key
details about the corresponding product:

> - product photo

Reviewer 2: Consider capitalizing these bullet points!

- Product title
- Price or cost of the product, depending on whether it is bought, sold, or used as a
  component
- Reference code (e.g. *DESK0005*)
- On-hand quantity
- Variant attributes (e.g. *Color: White*)

.. image:: product_catalog/product-card.png
   :align: center
   :alt: A product card in the product catalog.

> Products displayed on the page can be filtered using the search bar at the top of the page, or the

Reviewer: ```suggestion
Products can be filtered using the search bar at the top of the page, or the
```

sidebar on the left side of the page.

To filter by product type, click the :icon:`fa-caret-down` :guilabel:`(Toggle Search Panel)` button
on the right side of the search bar to open the search menu. In the :guilabel:`Filters` section,
select the :guilabel:`Services` filter to only show service products, or the :guilabel:`Products`
filter to only show physical products.

> When creating or configuring an |RfQ| or |SO| specifically, an :guilabel:`In the Order` filter

Reviewer 2: ```suggestion
When creating or configuring a quotation or |SO| specifically, an :guilabel:`In the Order` filter
```
